code/src/App.js

```diff
@@ -1,9 +1,85 @@
-import React from 'react'
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable react/react-in-jsx-scope */
```

Comment:
You can always edit `eslintrc.json` file in order to change some rules - it's cleaner approach!